src/pygeoprocessing/routing/routing.pyx

```diff
@@ -833,7 +812,7 @@ def fill_pits(
                 # search neighbors for downhill or nodata
                 downhill_neighbor = 0
                 nodata_neighbor = 0
-
+                neighbor_list = []

```

Comment:
Outside of appending to this list on 825, is this list supposed to be used elsewhere?

Reply:
Ahh yeah i used that for debugging. Removed inf1bb262